packages/fluentui/react-northstar/src/components/Chat/ChatItem.tsx

```diff
@@ -135,7 +135,7 @@ export const ChatItem = (React.forwardRef<HTMLLIElement, ChatItemProps>((inputPr
     });
 
     return (
-      <ChatItemContextProvider value={{ attached }}>
+      <ChatItemContextProvider value={{ attached, unstable_layout: layout }}>
```

Comment:
@Hirse this wasn't memoized before, so I just kept it as-is so that this PR only changes one thing. Will gladly make the change though if you'd like.